nemo/utils/timers.py

```diff
@@ -39,7 +39,6 @@ def __init__(self, reduction="mean", sync_cuda=False, buffer_size=-1):
         Args:
             reduction (str): reduction over multiple timings of the same timer
                              (none - returns the list instead of a scalar)
-            sync_cuda (bool): if True torch.cuda.synchronize() is called for start/stop
```

Comment:
Why remove description?

Reply:
Apologies, I think I messed up while fixing merge conflicts.

Reply:
Added this back.